rclcpp_examples/src/topics/listener_memory.cpp

```diff
@@ -23,9 +23,11 @@
 
 using rclcpp::memory_strategies::static_memory_strategy::StaticMemoryStrategy;
 
-void chatterCallback(const std_interfaces::msg::String::ConstSharedPtr & msg)
+unsigned int messages_received = 0;
+
+void chatterCallback(const std_interfaces::msg::String::ConstSharedPtr & /*msg*/)
 {
-  printf("I heard: [%s]\n", msg->data.c_str());
+  ++messages_received;
```

Comment:
I'm working towards making this example parallel to the examples I used for real-time testing, which avoid screen I/O because of the potential for nondeterministic blocking.